ansible/configs/sap-integration/files/repos_template.j2

```diff
@@ -0,0 +1,29 @@
+[rhel-7-server-rpms]
```

Comment:
What are you using this for? You have RHEL 8 instances and should be using satellite.

Reply:
done